apps/frontend/src/pages/auth/components/sign-up-form/styles.module.css

```diff
@@ -1,5 +1,76 @@
-.form {
+.auth-container {
+	display: flex;
+	width: 100vw;
+	height: 100vh;
```

Comment:
avoid using vh, let's try to achieve it using 100% or flex
```suggestion
```

Reply:
I'm not sure why, but it's not working as expected.
The .auth-container has a height of 100%, but the .left-side and .right-side are not taking the full height.
Here is the code for the container:
```
.auth-container {
	display: flex;
	flex-grow: 1;
	align-items: stretch;
	width: 100%;
	height: 100%;
}
```
And for the left and right sides:
```
.left-side,
.right-side {
	display: flex;
	width: 50%;
	height: 100%;
}
```
I've also verified body and root:
```
body {
	display: flex;
	flex-direction: column;
	min-width: 320px;
	min-height: 100vh;
	margin: 0;
	font-family: Inter, Arial, sans-serif;
	font-size: 14px;
	line-height: 1.5;
}

#root {
	display: flex;
	flex-direction: column;
	flex-grow: 1;
}
```

Reply:
Maybe your trained eye already know the answer :sweat_smile: